retrofit2/build.gradle

```diff
@@ -0,0 +1,13 @@
+archivesBaseName = 'armeria-retrofit2'
+
+dependencies {
+    compile project(':core')
+    compile 'com.google.guava:guava'
+    compile 'com.squareup.retrofit2:retrofit:2.1.0'
+
+    testCompile 'junit:junit'
+    testCompile 'com.squareup.okhttp3:mockwebserver:3.4.0'
+    testCompile 'com.squareup.retrofit2:adapter-guava:2.1.0'
+    testCompile 'com.squareup.retrofit2:converter-jackson:2.1.0'
```

Comment:
Could you add all dependency versions to the `dependencyManagement` section of the parent `build.gradle`?